jmix-templates/content/project/application/src/main/resources/${project_rootPath}/view/user/user-list-view.xml

```diff
@@ -2,11 +2,11 @@
 <view xmlns="http://jmix.io/schema/flowui/view"
       title="msg://UserListView.title"
       focusComponent="usersDataGrid">
-    <data readOnly="true">
+    <data>
```

Comment:
The same change is needed for the Kotlin project template

Reply:
Done